docs/brain-teasers/sumSquareDifference.md

```diff
@@ -0,0 +1,45 @@
+# (I) `sumSquareDifference`
+
+```javascript
+/**
+ * sumOfSquares
+ * @description
+ * The sum of the squares of the first ten natural numbers is, 1^2 + 2^2 + ... + 10^2 = 385.
+ * The square of the sum of the first ten natural numbers is, ( 1 + 2 + ... + 10 )^2 = 55^2 = 3025.
+ * The difference between the sum of the squares of the squares of the first ten natural numbers and the square of the sum is 3025 - 385 = 2640.
+ * Find the difference between the sum of the squares of the first n natural numbers and the square of the sum.
+ * @param {number} n - number to create range
+ * @return {number} - difference between the sum of the squares of the first n natural numbers and the square of the sum
+ */
+
+function sumSquareDifference( n ) {
+    
+}
+
+console.assert( sumSquareDifference( 100 ) );
```

Comment:
`assert` can only evaluate a statement as being `true` or `false` - and the assert fails if the condition is `false`

Please modify to reflect something like
`console.assert( sumSquareDifference( 100 ) === 25164150 );`